pallets/pallet-bonded-coins/src/lib.rs

```diff
@@ -147,13 +148,27 @@ pub mod pallet {
 	#[pallet::event]
 	#[pallet::generate_deposit(pub(super) fn deposit_event)]
 	pub enum Event<T: Config> {
-		Todo,
+		/// A bonded token pool has been moved to destroying state. [pool_id]
+		DestructionStarted(T::PoolId),
```

Comment:
It's better to use structs for events, so that each field is named.

Reply:
We should probably introduce a struct for this in a PR that's merged before this one

Reply:
No I mean you can do `DestructionStarted { id: T::PoolId }` directly here. No need for a new struct definition.

Reply:
15932d8c12aa37b3ce7eb338f202e4cf2ab872d0